app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -105,6 +106,10 @@ class Core(private val context: Context) {
             // Install the "icons" WebExtension to automatically load icons for every visited website.
             icons.install(engine, sessionManager)
 
+            // Show an ongoing notification when recording devices (camera, microphone) are used by web content
+            RecordingDevicesNotificationFeature(context, sessionManager)
```

Comment:
I don't like this API (constructing objects and not storing them anywhere in this scope). But that feedback isn't for this PR.

Reply:
Yeah, me neither. But I'm not sure how to improve. This requires a snippet of code that observes our sessions and nobody to keep a reference to it. The `enable()` only exists because doing all that from the constructor would be even crazier. :)

Reply:
In another case I have hidden that behind an extension method, something like:

`fun SessionManager.enableRecordingDevicesNotification()`

... but this only looks better and it's the same behind the facade.